src/app/profile/settings/connected-accounts/connected-accounts.component.html

```diff
@@ -0,0 +1,52 @@
+<div class="container-fluid container-cards-pf">
+  <div class="row-cards-pf accounts-component">
+    <div class="col-xs-10 col-sm-4 col-md-4">
+      <div class="card-pf">
+        <div class="card-pf-heading">
+          <div class="row account-content">
+            <div class="col-xs-6 col-sm-6 col-md-6">
```

Comment:
No need to define this as being 6 columns for each size. If you set it to just `col-xs-6`, then it will fill 6 columns for all media queries.